### ISSUE TITLE
Secion for Resources

### ISSUE THREAD

**Issue by whiskeysierra** [terraform-docs/terraform-docs#276]
**Is your feature request related to a problem? Please describe.**

I'm often missing the list of resources that are being created by a module.
It would help to get a feeling of what a module provides. 
That can and should (to some degree) be done manually in plain english.
But having a generated list gives me more confidence as a reader that I know the full picture.

**Describe the solution you'd like**

Next the inputs and outputs I'd like to have a resources section, listing all resources (by type, maybe count, if).

**Describe alternatives you've considered**

- describe it in prose in the README
- looking into the source code directly

**Additional context**

Possible output:

## Resources

| Type                    | Count |
|-------------------------|-------|
| azurerm_virtual_network | 1     |
| azurerm_subnet          | 0..1  |

https://registry.terraform.io/modules/Azure/network-security-group/azurerm/3.0.1?tab=resources

**Comment by khos2ow** [comment]
This is a very good suggestion @whiskeysierra and I've been thinking about it too and similar features have been requested before (e.g. dependant modules, output type, local_file, etc) which at the moment none of them are possible due to limitation of `terraform-config-inspect` (the way we extract information).

I've started a PoC of expanding terraform-config-inspect to be able to do all of these without adding a big overhead of complexity and execution times, but it's not complete yet. I'll be able to revisit this as soon as I ironed out that.

**Comment by nitrocode** [comment]
I'd love to see if those resources are also able to be disabled individually. For instance, I was vetting the [cloudposse/terraform-aws-ecs-web-app](https://github.com/cloudposse/terraform-aws-ecs-web-app) module and saw that it uses code pipeline which I don't need and luckily it can be disabled using `var.codepipeline_enabled`

| Type | Name | Count | Variable to enable/disable ? |
| ------------- | ------------- | ------------- | ------------- |
| azurerm_virtual_network | this | 1 | No |
| azurerm_subnet | this | 0..1 | `var.enable_subnets` |

**Comment by khos2ow** [comment]
For some reason we've missed to mark #287 as duplicate of this issue and most of the discussion and traction around this have been happening on the other issue. Good news, though, there is an open PR for it: #347!
